docs/api_call.md

```diff
@@ -1,1352 +1,1108 @@
-            }
-          },
-          {
-            "detection": "message",
-            "original_text": "dominos",
-            "entity_value": {
-              "value": "Domino's Pizza"
-            }
-          }
-        ]
-      }
-      ```
-
-  - Example 2: 
-
-    - ```python
-      message = 'i wanted to watch movie'
-      entity_name = 'movie'
-      structured_value = 'inferno'
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:* 
-
-      ```python
-      from ner_v1.chatbot.entity_detection import get_text
-      output = get_text(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20wanted%20to%20watch%20movie&entity_name=movie&structured_value=inferno&fallback_value=None&bot_message='
-      ```
-
-    - *CURL Output*:
-
-    - ​
-
-      ```json
-      {
-        "data": [
-          {
-            "detection": "structure_value_verified",
-            "original_text": "inferno",
-            "entity_value": {
-              "value": "Inferno"
-            }
-          }
-        ]
-      }
-      ```
-
-  - ​
-
-
-### phone_number
+2. **entity_name**: Name of the entity. This parameter is important when the detection logic relies on dictionary. It is used to run particular detection logic by looking at its dictionary.
 
-- This functionality calls the PhoneDetector class to detect textual entities.
+   For example, in Text detection logic  the entity name is based on dictionary name of entities we need to detect.
 
-- Example:
+   - For detecting:
+     - *cuisine* entity_name will be *"cuisine"*
+     - *dish* entity_name will be *"dish"*
 
-  - Example 1: 
+   In some detection like `date` or `time detection` `entity_name` does not have any siginificance as detection logic of these entities won't change based on their name.
 
-    - ```python
-      message = 'my contact number is 9049961794'
-      entity_name = 'phone_number'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:* 
+3. **structured_value**: It is a value which is obtained from the structured text. For example, UI elements like form, payload, etc.
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_phone_number
-      output = get_phone_number(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
+4. **fallback_value**: It is a fallback value. If the detection logic fails to detect any value either from *structured_value* or *message* then we return a *fallback_value* as an output. This value is derived from third party api or from users profile.
```

Comment:
Again simplification and better explanation needed here. Sentence "This value is derived from third party api or from users profile." makes no sense and has no context here.